### ISSUE TITLE
No `search count` as virtual text if `keymaps.basic = true` (Conflict with noice.nvim)

### ISSUE THREAD

**Issue by ilias777** [declancm/cinnamon.nvim#50]
If basic keymaps is set to `true`, the `search count` as virtual text don't appear any more by searching for any text. This virtual search count comes from noice.nvim.

**With `keymaps.basic = false`:**

<img width="470" alt="without-basic" src="https://github.com/declancm/cinnamon.nvim/assets/39212564/f91bb001-45f5-4998-82f5-b40756113df8">

***With `keymaps.basic = true`:**

<img width="441" alt="with-basic" src="https://github.com/declancm/cinnamon.nvim/assets/39212564/57f169ca-3084-4cba-8abd-86ca1ef40c94">

**Comment by declancm** [comment]
I'm not sure yet about the best way of fixing this.

A workaround for now could be:

```lua
vim.keymap.set('n', 'n', function()
  cinnamon.scroll('n', { callback = vim.schedule_wrap(function()
    vim.cmd('norm! Nn')
  end) })
end)
vim.keymap.set('n', 'N', function()
  cinnamon.scroll('N', { callback = vim.schedule_wrap(function()
    vim.cmd('norm! nN')
  end) })
end)

```

**Comment by declancm** [comment]
Ah the issue is that the search count messages don't work when lazyredaw is set. This isn't just a conflict with noice. I'll hopefully have a fix soon!

**Comment by ilias777** [comment]
Thank you 🙂

**Comment by ilias777** [comment]
Unfortunately the issue still exist.
Do I have to change some settings?

Look:

https://github.com/user-attachments/assets/65a956aa-d078-4906-a0f1-2d134a70b61c

Only if basic keymaps are enabled.
I have of course the latest version of cinnamon `1.0.3`.

**Comment by declancm** [comment]
It should be fixed if you remove the `version = "*"` :). I haven't released a new version with this fix yet as I'm working towards a v1.1.0

**Comment by ilias777** [comment]
Ok. I have removed the version flag. I tried it out. It works, but not always. Sometimes the search count appears normal as aspected, sometimes in the wrong place and sometimes not at all.

Here a test:

https://github.com/user-attachments/assets/7a482e08-239b-46f5-9b76-9318115a56d1

It's good that you know how to fix it.
Thank you for your plugin. I love it 😊

I'll be waiting for the version 1.1.0 with the fix. 🙂
